### PR TITLE
HL-NFX-0027 - Add initial modal and full sound autoplay

### PR DESCRIPTION
**Pull request by galias11**

### CODE REVIEW

File: src/components/modal/Modal.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+            src={micLogo}
+            alt="healiner microphone"/>
+          <p className={styles.textWrapper}>
+            <FormattedText
+              className={styles.welcomeText}
+              textKey="introModal-welcome"/>
+            <FormattedText
+              textKey="introModal-modalText"/>
+          </p>
+          <HeadilnerButton
```
Comment (Ruk33) [inline src/components/modal/Modal.tsx]: Gerar you have a typo here.

File: src/reducers/player.reducer.ts [hunk 2]
```diff
@@ -11,11 +12,15 @@ import { Action } from '../constants/types';
...
-  currentPlayingId: undefined
+  currentPlayingId: undefined,
+  playingEnabled: false
 });
 
 const slidersReducer = produce((nextState: PlayerState, action: Action): PlayerState => {
   switch(action.type) {
+    case PLAYER_ENABLE:
+      nextState.playingEnabled = true;
+      return nextState;
```
Comment (Ruk33) [inline src/reducers/player.reducer.ts]: Gerar this is mutating the state, you should create a copy with the new values instead.

File: src/components/modal/Modal.tsx (hunk 1 above)
Comment (galias11) [reply src/components/modal/Modal.tsx]: Done, please recheck

File: src/reducers/player.reducer.ts (hunk 2 above)
Comment (galias11) [reply src/reducers/player.reducer.ts]: As we talked,  immer (produce) manages state immutability with a proxy

Comment (Ruk33) [reply src/reducers/player.reducer.ts]: Thank you Gerar, you're right https://immerjs.github.io/immer/docs/produce, this is awesome!

**Pull request merged**